utkdigitalinitiatives/islandora_bagit
# add check for PRESERVE and switch OBJ  in keeper array

**JIRA Ticket**: https://jirautk.atlassian.net/browse/DIGITAL-1294

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)

# What does this Pull Request do?
   This modifies the ds_utk plugin  to test for the PRESERVE datastream  in the current object being bagged, and if it is there, replace the "OBJ" element in the array of datastreams to keep.

# What's new?
   The previous test just compared the keepers array to each datastream name as it  listed the datastreams in the object and allowed it to be saved to the final array if it was in the string.  A new test was put in to check for the PRESERVE.

# How should this be tested?
   You will have to set up bagit for your own user in the islandora gui, then select several pids that have no OBJ or PRESERVE and an OBJ and an object with OBJ and PRESERVE datastreams.
  Make a bagit bag of those objects.   The first kind of object should have neither OBJ or PRESERVE datastreams saved, the other kinds should have one or the other but not both. 

# Additional Notes:
  The easiest way to test this is to make bags on digital and then make them on porter.  Porter already has this in place.

# Interested parties
@utkdigitalinitiatives/digital-initiatives-developers
